### ISSUE TITLE
kube_service kube_deployment modules not copied before shipit

### ISSUE THREAD

**Issue by nonotest** [ansible/ansible-container#174]
##### ISSUE TYPE
 - Bug Report

##### OS / ENVIRONMENT
```
$ ansible-container --debug version
Ansible Container, version 0.1.0
Darwin, AM-MBP.local, 15.6.0, Darwin Kernel Version 15.6.0: Thu Jun 23 18:25:34 PDT 2016; root:xnu-3248.60.10~1/RELEASE_X86_64, x86_64
2.7.12 (default, Jun 29 2016, 14:05:02)
[GCC 4.2.1 Compatible Apple LLVM 7.3.0 (clang-703.0.31)] /usr/local/opt/python/bin/python2.7
No DOCKER_HOST environment variable found. Assuming UNIX socket at /var/run/docker.sock
{u'Architecture': u'x86_64',
 u'BridgeNfIp6tables': True,
 u'BridgeNfIptables': True,
 u'CPUSet': True,
 u'CPUShares': True,
 u'CgroupDriver': u'cgroupfs',
 u'ClusterAdvertise': u'',
 u'ClusterStore': u'',
 u'Containers': 1,
 u'ContainersPaused': 0,
 u'ContainersRunning': 0,
 u'ContainersStopped': 1,
 u'CpuCfsPeriod': True,
 u'CpuCfsQuota': True,
 u'Debug': True,
 u'DefaultRuntime': u'runc',
 u'DockerRootDir': u'/var/lib/docker',
 u'Driver': u'aufs',
 u'DriverStatus': [[u'Root Dir', u'/var/lib/docker/aufs'],
                   [u'Backing Filesystem', u'extfs'],
                   [u'Dirs', u'41'],
                   [u'Dirperm1 Supported', u'true']],
 u'ExecutionDriver': u'',
 u'ExperimentalBuild': True,
 u'HttpProxy': u'',
 u'HttpsProxy': u'',
 u'ID': u'VCNW:FUC5:KZX7:3QHY:RGOU:EWZT:Y5M3:PC2V:DC3E:ABUY:GCE4:JZFT',
 u'IPv4Forwarding': True,
 u'Images': 9,
 u'IndexServerAddress': u'https://index.docker.io/v1/',
 u'KernelMemory': True,
 u'KernelVersion': u'4.4.16-moby',
 u'Labels': None,
 u'LiveRestoreEnabled': False,
 u'LoggingDriver': u'json-file',
 u'MemTotal': 8362258432,
 u'MemoryLimit': True,
 u'NCPU': 4,
 u'NEventsListener': 1,
 u'NFd': 18,
 u'NGoroutines': 29,
 u'Name': u'moby',
 u'NoProxy': u'*.local, 169.254/16',
 u'OSType': u'linux',
 u'OomKillDisable': True,
 u'OperatingSystem': u'Alpine Linux v3.4',
 u'Plugins': {u'Authorization': None,
              u'Network': [u'bridge', u'host', u'null', u'overlay'],
              u'Volume': [u'local']},
 u'RegistryConfig': {u'IndexConfigs': {u'docker.io': {u'Mirrors': None,
                                                      u'Name': u'docker.io',
                                                      u'Official': True,
                                                      u'Secure': True}},
                     u'InsecureRegistryCIDRs': [u'127.0.0.0/8'],
                     u'Mirrors': None},
 u'Runtimes': {u'runc': {u'path': u'docker-runc'}},
 u'SecurityOptions': [u'seccomp'],
 u'ServerVersion': u'1.12.0',
 u'SwapLimit': True,
 u'Swarm': {u'Cluster': {u'CreatedAt': u'0001-01-01T00:00:00Z',
                         u'ID': u'',
                         u'Spec': {u'CAConfig': {},
                                   u'Dispatcher': {},
                                   u'Orchestration': {},
                                   u'Raft': {},
                                   u'TaskDefaults': {}},
                         u'UpdatedAt': u'0001-01-01T00:00:00Z',
                         u'Version': {}},
            u'ControlAvailable': False,
            u'Error': u'',
            u'LocalNodeState': u'inactive',
            u'Managers': 0,
            u'NodeAddr': u'',
            u'NodeID': u'',
            u'Nodes': 0,
            u'RemoteManagers': None},
 u'SystemStatus': None,
 u'SystemTime': u'2016-08-15T06:02:32.645995499Z'}
{u'ApiVersion': u'1.24',
 u'Arch': u'amd64',
 u'BuildTime': u'2016-07-28T21:04:48.864474905+00:00',
 u'Experimental': True,
 u'GitCommit': u'8eab29e',
 u'GoVersion': u'go1.6.3',
 u'KernelVersion': u'4.4.16-moby',
 u'Os': u'linux',
 u'Version': u'1.12.0'}
```
##### SUMMARY
Hello,

I have ran into some strange issues with the shipit command:

It looks like the step responsible to copy the kube_service etc modules in the kubernetes role library before shipit didn't do anything.

As a result I had to manually copy the files to be able to complete the shipit command.

Someone reported a similar issue there:
https://groups.google.com/forum/#!topic/ansible-devel/D6Sto4NH1Fc

I am running ansible-container on macos el capitan with pyton installed from homebrew and pip.
let me know if you need more details?

##### STEPS TO REPRODUCE
```
ansible-playbook shipit-kubernetes.yml
```

##### EXPECTED RESULTS
````
ansible-playbook shipit-kubernetes.yml

PLAY [Deploy test to  kubernetes] *******************************************

TASK [test-kubernetes : kube_service] ***************************************
ok: [localhost]

TASK [test-kubernetes : debug] **********************************************
skipping: [localhost]

TASK [test-kubernetes : kube_service] ***************************************
ok: [localhost]

TASK [test-kubernetes : debug] **********************************************
skipping: [localhost]

TASK [test-kubernetes : kube_service] ***************************************
ok: [localhost]

TASK [test-kubernetes : debug] **********************************************
skipping: [localhost]

TASK [test-kubernetes : kube_deployment] ************************************
changed: [localhost]

TASK [test-kubernetes : debug] **********************************************
skipping: [localhost]

TASK [test-kubernetes : kube_deployment] ************************************
changed: [localhost]

TASK [test-kubernetes : debug] **********************************************
skipping: [localhost]

TASK [test-kubernetes : kube_deployment] ************************************
changed: [localhost]

TASK [test-kubernetes : debug] **********************************************
skipping: [localhost]

PLAY RECAP *********************************************************************
localhost                  : ok=6    changed=3    unreachable=0    failed=0
````

##### ACTUAL RESULTS

````
$ ansible-playbook shipit-kubernetes.yml
ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '.../ansible/roles/test-kubernetes/tasks/main.yml': line 2, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- kube_service:
  ^ here


The error appears to have been in '..../ansible/roles/test-kubernetes/tasks/main.yml': line 2, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- kube_service:
  ^ here
````

**Comment by chouseknecht** [comment]
Think I finally figured out what the issue was. If you installed using *pip*, the Ansible modules were not being installed, and there was nothing to copy into the *library* folder. This is fixed in 0.2.0. To install 0.2.0 you can do the following:

```
pip install --upgrade  git+https://github.com/ansible/ansible-container.git
```

Or, you can clone the repo and run *setup* as described in the [Installation Guide](http://docs.ansible.com/ansible-container/installation.html).

If you are still unable to get things working or have additional questions, please feel free to re-open this issue.

**Comment by nonotest** [comment]
yep works now. thanks !
